Airbnb.APIs/Program.cs

```diff
@@ -17,7 +17,7 @@ public static void Main(string[] args)
             builder.Services.AddSwaggerGen();
             builder.Services.AddDbContext<AirbnbDbContext>(options =>
             {
-                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"));
+                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"),b=>b.MigrationsAssembly("Airbnb.APIs"));
```

Comment:
There is no need to create another migrations folder, which exists inside infrastructure/Data